resources/baseJenkins.yaml

```diff
@@ -175,8 +175,16 @@ tool:
       properties:
       - installSource:
           installers:
-          - zip:
-              url: "https://ci.opensearch.org/ci/dbc/tools/OpenJDK14U-jdk_x64_linux_hotspot_14.0.2_12.zip"
+          - command:
+              command: "JENKINS_HOME_LINUX=\"/var/jenkins\"\nJENKINS_HOME_WINDOWS=\"\
+                C:/Users/Administrator/jenkins\"\n \n\nif uname -s | grep -i NT; then\n\
+                \    echo windows agent\n    JENKINS_JDK14_PATH=$JENKINS_HOME_WINDOWS/tools/hudson.model.JDK/openjdk-14\n\
+                \    JDK14_ZIP=\"OpenJDK14U-jdk_x64_windows_hotspot_14.0.2_12.zip\"\
+                \n    \nelse\n     echo linux agent\n     JENKINS_JDK14_PATH=$JENKINS_HOME_LINUX/tools/hudson.model.JDK/openjdk-14\n\
+                \     JDK14_ZIP=\"OpenJDK14U-jdk_x64_linux_hotspot_14.0.2_12.zip\"\
+                \n     \nfi\n\n\nmkdir -p $JENKINS_JDK14_PATH\ncd $JENKINS_JDK14_PATH\n\
+                pwd\nrm -rf ./*\ncurl -sSL https://ci.opensearch.org/ci/dbc/tools/$JDK14_ZIP\
```

Comment:
Why are we using ci.opensearch.org? We are trying to keep this ci as generic as possible avoid any urls/artifacts related to OpenSearch

Reply:
[Offline discussion] We will be creating an issue to take this input from user rather than hardcoding all the commands. Something similar to https://github.com/opensearch-project/opensearch-ci/pull/136/files

Reply:
There are some modifications on the zip so it can install without further tweaks.
I will switch from ci to adoptium links and will open an issue to add a custom entry for tweaking global tools.

Ex: https://github.com/opensearch-project/opensearch-ci/pull/136
Thanks.